plugins/rbac/src/components/CreateRole/RoleDetailsForm.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+import { TextField } from '@material-ui/core';
+import { FormikProps } from 'formik';
+
+import { CreateRoleFormValues } from './types';
+
+export const RoleDetailsForm = ({
+  formik,
```

Comment:
Here is another example where I would suggest not accepting the formik instance but instead accept the specific properties.   Let the parent component deal with the formik instance, this component then becomes much simpler and potentially usable with other form libraries even.